rlclientlib/utility/config_utility.cc

```diff
@@ -1,18 +1,17 @@
 #include <map>
 #include <set>
+#include <rapidjson/document.h>
+#include <rapidjson/prettywriter.h>
 #include <regex>
 #include "config_utility.h"
-#include "cpprest/json.h"
 #include "configuration.h"
 #include "constants.h"
 #include "err_constants.h"
 #include "api_status.h"
 #include "str_util.h"
 #include "trace_logger.h"
-#include "../model_mgmt/restapi_data_transport.h"
 
-using namespace web;
-using namespace utility::conversions; // string conversions utilities
+using namespace rapidjson;
```

Comment:
I'd prefer to either shorted this to `rj` or just use the full namespace.

Reply:
Done